code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1274,4 +1274,64 @@
 			M.adjustStaminaLoss(1.5*REM, 0)
 	..()
 	. = 1
-  
+
+
+/datum/reagent/medicine/oxycodone
+	name = "Oxycodone"
+	id = "oxycodone"
+	description = "An effective and very addictive painkiller."
+	metabolization_rate = 0.05 * REAGENTS_METABOLISM
+	color = "#C805DC"
+	taste_description = "bitterness"
+	addiction_threshold = 15
```

Comment:
no actual defined addiction steps

no downsides to this either, I can just ride high on oxy all fucking shift and nullify everything you did here

Reply:
it's supposed to be a warning like "hey dipshit you're getting close to overdose"